splice_machine/src/main/java/com/splicemachine/derby/utils/Vacuum.java

```diff
@@ -68,6 +68,23 @@ public Vacuum(Connection connection) throws SQLException {
         }
     }
 
+    private void addNewFlushFuture(List<Future> flushFutures,
+                                   long tableConglom,
+                                   String tableName,
+                                   PartitionFactory partitionFactory) {
+        flushFutures.add(SIDriver.driver().getExecutorService().submit(new Runnable() {
+            @Override
+            public void run() {
+                LOG.info("Flushing possibly-active dropped table: " + tableName);
+                try(Partition partition=partitionFactory.getTable(Long.toString(tableConglom))){
+                    partition.flush();
+                }catch(IOException e){
+                    LOG.error("Vacuum Unexpected exception", e);
```

Comment:
We probably don't want to add any extra retries of the flushing here.  Maybe this is a case where the table is being compacted or some other operation is temporarily locking out the flush operation, so it likely would fail again trying the operation so soon after the first attempt.  The next time vacuum is run, perhaps the temporary condition will be gone, and at that time the table could be flushed or deleted.

Reply:
That makes sense, thank you